### PR TITLE
fix: button style

### PR DESCRIPTION
**Pull request by ZanBin**
## Check List
- [ ] Your code builds clean without any errors or warnings.
- [ ] Check coding convention.
- [ ] Updated Storybook components and stories to reflect any changes made to the UI.
- [ ] Test covers all edge cases and coverage is greater than or equal to 80%.

## Proof of Completeness
Screenshot changes introduced" width="250" />
OR
Video link

### CODE REVIEW

**Review by loc-nguyenthien — CHANGES_REQUESTED** [review]

**Review by loc-nguyenthien — APPROVED** [review]

File: src/components/Button/Button.tsx [hunk 1]
```diff
@@ -131,9 +140,12 @@ const ButtonWrapper = styled.TouchableOpacity<Omit<ButtonProps, 'text' | 'onPres
...
 
-const Label = styled(Text)(({theme, color}) => ({
-  color,
-  fontWeight: theme?.fontWeights?.bold,
-}))
+const Label = styled(Text)(
+  ({theme, color, textSize}: {color?: string; theme?: ITheme; textSize?: number}) => ({
+    color,
+    fontWeight: theme?.fontWeights?.bold,
+    fontSize: textSize ? textSize : metrics.xs,
```
Comment (honghoangsts) [inline src/components/Button/Button.tsx:147]: `textSize ?? metrics.xs`

**Review by honghoangsts — CHANGES_REQUESTED** [review]

**Review by honghoangsts — APPROVED** [review]

**Review by honghoangsts — APPROVED** [review]
